spec/api/create-dashboard.spec.js

```diff
@@ -13,6 +13,8 @@ var newVisMock = function () {
   };
   visMock.dataviews = jasmine.createSpyObj('dataviews', ['createFormulaModel']);
   visMock.dataviews.createFormulaModel.and.returnValue(new Backbone.Model());
+  visMock.analysis = jasmine.createSpyObj('analysis', ['findNodeById']);
+  visMock.analysis.findNodeById.and.returnValue(new Backbone.Model());
   visMock.done = function (callback) { callback(); };
```

Comment:
What about updating the old mocks to jasmine?

```javascript
spyOn(visMock, 'analysis').and.callFake(function(callback) { callback()});
```

Reply:
ein? Please ellaborate.

Reply:
Using jasmine `spyOn` Instead creating mocks "manually" like this:

```javascript
 visMock.done = function (callback) { callback(); };
```

(is just a suggestion, not relevant al all :P )

Reply:
In this case I'm not sure it's worth it. The correspondent Jasmine code is more verbose

```js
visMock.done = jasmine.createSpy('done').and.callFake(function (callback) { callback(); });
```

and the code is not using anything we could take advantage of directly. It's creating spies on the callback function, not in `done` or `error`. It's not checking that `done` is called, but the callback it receives as an argument.